warehouse/ingest-core/pom.xml

```diff
@@ -163,11 +294,35 @@
             <version>${version.accumulo}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-core</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>org.javassist</groupId>
             <artifactId>javassist</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.23.0</version>
```

Comment:
Top level pom has `version.mockito` at 1.10.19....maybe we should upgrade that to 2.23.0?

Reply:
+1  i would just remove the version and use the version managed by the parent.